src/oralb_ble/parser.py

```diff
@@ -69,7 +69,7 @@ class ModelDescription:
 }
 
 
-SMART_SERIES_6000_MODES = SMART_SERIES_4000_MODES = SMART_SERIES_MODES | {2: "off"}
+SMART_SERIES_6000_MODES = SMART_SERIES_MODES | {2: "off"}
```

Comment:
I'm not too experienced with Python to know for sure if this is correct.

Reply:
I assumed thats what is was based on your report here
https://github.com/home-assistant/core/issues/81489

But if this fixes it for you I'm happy to merge it

Reply:
Install the library in a dev environment and startup home assistant to be sure.


If you don't have one you can install it manually inside your production container using

https://developers.home-assistant.io/docs/operating-system/debugging/

Or disabling protected mode in the Community ssh add on and entering the container with

docker exec -it homeassistant /bin/bash

Reply:
Inside the docker container, I made the same changes in `/usr/local/lib/python3.10/site-packages/oralb_ble/parser.py` and my fix works fine.

Even better, I now understand the confusion I had before in https://github.com/home-assistant/core/issues/81489.
When I turn the device off, I need to hold the button. The toothbrush does switch to the next mode when doing this. I've tested this by also turning it off while in `sensitive` and it will first go to `whitening ` before turning off. I can even hear this as the toothbrush makes a different noise.

Sorry for the confusion in the first place :-)

Reply:
So good to merge this?

Reply:
Yes!

Reply:
You'll need to open a Pr to bump this library in Home assistant as well once the tool generates a new release in a minute or two

Reply:
The tooling rejected the commit message and didn’t generate a release since the title didn’t start with a lowercase char.

If you open an empty pr with the title ‘fix: release’ I’ll merge it and we will get one.

otherwise I can fix it once I get home in a few hours

Reply:
I noticed that when I created the PR, I had adjusted the commit message and pushed that, but apparently, that didn't work.

Reply:
Apparently that fix didn't make it to this PR and remained in my fork: https://github.com/golles/oralb-ble/commit/55d38da4ed59858362d22ec6e0b5a51a42cac7e9

Reply:
Guess we need a fresh pr anyways 😉